ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -49,6 +49,7 @@ StatusDialog {
     property var nestedCollectiblesModel: store.nestedCollectiblesModel
     property var bestRoutes
     property bool isLoading: false
+    property int loginType
```

Comment:
Not sure about this name here.. On this context, I think it should be something like this:

```suggestion
    property int authenticationType
```

Reply:
Why, it's called `loginType` everywhere else, this would only imho create confusion. Although I agree `authenticationType` is a better name :shrug:

Reply:
As everywhere else, it means how the user logs in (password, biometrics, keycard).

RootStore.qml:
```qml
    readonly property int loginType: getLoginType()
    function getLoginType() {
        if(!userProfileInst)
            return Constants.LoginType.Password

        if(userProfileInst.usingBiometricLogin)
            return Constants.LoginType.Biometrics
        if(userProfileInst.isKeycardUser)
            return Constants.LoginType.Keycard
        return Constants.LoginType.Password
    }
```

then it's propagated into SendModal.qml:
```qml
sourceComponent: SendModal {
    onlyAssets: sendModal.onlyAssets      
    store: appMain.transactionStore
    loginType: appMain.rootStore.loginType
```

Reply:
This is the overall usage:
![image](https://github.com/status-im/status-desktop/assets/5377645/80c75cd6-3857-48ee-b188-b7b245c4f407)

Reply:
OK, the concensus was to rename to "authenticationType` only in this component

Reply:
I think it's fine as-is, the footer doesn't (and shouldn't imo) know anything about the logic, it just gets an icon name